R/helpers_data.R

```diff
@@ -207,3 +207,34 @@ h_obtain_dose_grid_range <- function(object, ignore_placebo) {
     range(dose_grid)
   }
 }
+
+#' Convert a Ordinal Data to the Equivalent Binary Data for a Specific
+#' Grade
+#'
+#' @description `r lifecycle::badge("experimental")`
+#'
+#' A simple helper function that takes a [`DataOrdinal`] object and an
+#' integer grade and converts them to the equivalent `Data` object.
+#'
+#' @param d (`DataOrdinal`)\cr the `DataOrdinal` object to covert
+#' @param grade (`integer`)\cr the toxicity grade for which the equivalent data
+#' is required.
+#' @return A [`Data`] object.
+#'
+#' @export
+h_convert_ordinal_data <- function(d, grade) {
```

Comment:
generally one letter argument names are not ideal (if not super standard like `x` etc.), how about `data`?

Reply:
Fair point.  I dislike `data` because of potential confusion with the function.  Shall we compromise on `df`?

Reply:
yeah sounds good